svg/struct/scripted/autofocus-attribute.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml">
+    <title>Autofocus attribute</title>
+    <metadata>
+        <h:link rel="help" href="https://svgwg.org/svg2-draft/struct.html#autofocusattribute"/>
```

Comment:
autofocus is longer mentioned in the SVG 2 spec.

Otherwise, test if fine.

Reply:
I've just merged the PR, the [spec reference is live now](https://svgwg.org/svg2-draft/struct.html#autofocusattribute). (Although w3.org seems to be down, so it's unstyled HTML right now!)